cores/arduino/stm32/usb/cdc/usbd_cdc_if.c

```diff
@@ -169,6 +170,7 @@ static int8_t USBD_CDC_Control(uint8_t cmd, uint8_t *pbuf, uint16_t length)
     case CDC_SET_CONTROL_LINE_STATE:
       lineState =
         (((USBD_SetupReqTypedef *)pbuf)->wValue & 0x01) != 0; // Check DTR state
+    hostPortOpen = lineState;

```

Comment:
Anyway, there is an indent issue:
```suggestion
      hostPortOpen = lineState;
```